pages/+layout.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
+	import '@evidence-dev/tailwind/fonts.css';
+	import '../app.css';
+	import { EvidenceDefaultLayout } from '@evidence-dev/core-components';
+	export let data;
+</script>
+
+<EvidenceDefaultLayout {data} title="Minor League Esports" logo=/MLE4.png githubRepo=https://github.com/Minor-League-Esports/evidence xProfile=https://twitter.com/MLEsportsGG >
```

Comment:
Remove the "title" element since it can't/won't be displayed if we have a logo (and I'll get confused about what it does later)